--- origsrc/glitz-0.5.6/configure.in	2006-05-20 15:29:43.000000000 -0500
+++ src/glitz-0.5.6/configure.in	2009-03-15 00:15:51.750000000 -0500
@@ -10,18 +10,18 @@ GLITZ_VERSION=0.5.6
 # libtool shared library version
 
 # Increment if the interface has additions, changes, removals.
-LT_CURRENT=1
+GLITZ_LT_CURRENT=1
 
 # Increment any time the source changes; set to
 # 0 if you increment CURRENT
-LT_REVISION=0
+GLITZ_LT_REVISION=0
 
 # Increment if any interfaces have been added; set to 0
 # if any interfaces have been removed. removal has
 # precedence over adding, so set to 0 if both happened.
-LT_AGE=0
+GLITZ_LT_AGE=0
 
-VERSION_INFO="$LT_CURRENT:$LT_REVISION:$LT_AGE"
+VERSION_INFO="$GLITZ_LT_CURRENT:$GLITZ_LT_REVISION:$GLITZ_LT_AGE"
 AC_SUBST(VERSION_INFO)
 
 dnl ===========================================================================
